cmd/bitxhub/start.go

```diff
@@ -156,22 +156,22 @@ func runtimePProf(mode string, id uint64, duration time.Duration) {
 
 	var cpuFile *os.File
 	if mode == "cpu" {
-		cpuPath := fmt.Sprint(rootPath,"cpu-", time.Now().Format("20060102-15:04:05"))
+		cpuPath := fmt.Sprint(rootPath, "cpu-", time.Now().Format("20060102-15:04:05"))
 		cpuFile, _ = os.Create(cpuPath)
 		_ = pprof.StartCPUProfile(cpuFile)
 	}
 	for {
 		select {
-		case <- tick.C:
+		case <-tick.C:
 			switch mode {
 			case "cpu":
 				pprof.StopCPUProfile()
 				_ = cpuFile.Close()
-				cpuPath := fmt.Sprint(rootPath,"cpu-", time.Now().Format("20060102-15:04:05"))
+				cpuPath := fmt.Sprint(rootPath, "cpu-", time.Now().Format("20060102-15:04:05"))
 				cpuFile, _ := os.Create(cpuPath)
 				_ = pprof.StartCPUProfile(cpuFile)
 			case "memory":
-				memPath := fmt.Sprint(rootPath,"mem-", time.Now().Format("20060102-15:04:05"))
+				memPath := fmt.Sprint(rootPath, "mem-", time.Now().Format("20060102-15:04:05"))
```

Comment:
please change to filepath.join
subPath := fmt.Sprint("mem-", time.Now().Format("20060102-15:04:05"))
memPath := filepath.Join(rootPath,subPath)